core_backend/app/urgency_detection/models.py

```diff
@@ -67,15 +67,15 @@ class UrgencyResponseDB(Base):
     Urgency response database model.
     """
 
-    __tablename__ = "urgency_responses"
+    __tablename__ = "urgency~responses"
```

Comment:
Oops I think both table names ended up with `~` in them instead of `-`!

Reply:
Yeah! Thanks for picking that up.
I noticed that as well. Didn't get time to fix it - will do after kids birthday party #2 

It's not ready for review as yet.